galaxy.yml

```diff
@@ -2,7 +2,7 @@
 
 namespace: cloud
 name: terraform
-version: 1.1.0
+version: 2.0.0
```

Comment:
I guess you can remane the version to 3.0.0-dev0 as we do in amazon https://github.com/ansible-collections/amazon.aws/blob/main/galaxy.yml#L4

Reply:
done